homeassistant/components/accuweather/__init__.py

```diff
@@ -104,9 +104,11 @@ async def _async_update_data(self) -> dict[str, Any]:
                 )
         except (
             ApiError,
+            ClientError,
             ClientConnectorError,
             InvalidApiKeyError,
             RequestsExceededError,
+            InvalidCoordinatesError,
```

Comment:
Please only change one integration per PR. Based on the name of the PR I assume the intention is to target bme680?

Reply:
Yes, this commit was in the same branch. My bad. Can you remove this commit?

Reply:
@MartinHjelmare It included the previous commit as well, which was reviewed already and cancelled. Let me know if the PR with the changes on BME680 files are fine.

Reply:
Please rebase your branch and remove the unrelated commits.